### ISSUE TITLE
Problem of reproducing your results

### ISSUE THREAD

**Issue by xihuachen9804** [VIPSeg-Dataset/VIPSeg-Dataset#3]
Thanks for opensourcing your code. However, when I tried to reproduce the code following "README.md", there are some problems.
We noticed that there are three datasets registered in your code, cityscapes_vps_image,cityscapes_vps_video and panovspw_vps_video, but none of them match the VIPSeg Dataset provided in the article. We tried to annotate the code of register_cityscapes_vps_image and register_cityscapes_vps_video in "ClipPanoFCN/panopticfcn/data/datasets/__init__.py", and changed the path of vpsw_video to the path of the VIPSeg data in "register_panovspw_vps_video.py".Although the eval process can be carried out in this way and we used the "model_final.pth" you provided in "README.md", the obtained results are quite different from the results proposed in the paper.

**Comment by VIPSeg-Dataset** [comment]
Sorry to make you confused. The path in panovspw_vps_video should be changed to the path of the VIPSeg dataset. Would you please show the results you obtained?

**Comment by xihuachen9804** [comment]
> Sorry to make you confused. The path in panovspw_vps_video should be changed to the path of the VIPSeg dataset. Would you please show the results you obtained?

Thanks for your response. We run run_eval.sh to eval the net. In this process, we changed DETECTRON2_DATASETS to  VIPSeg_720P , the DATAROOT  to VIPSeg_720P/panomasksRGB and the PRETRAIN_WEIGHT to "model final.pth" provided in "README.md".The others are not modified. 
 We obtained the results as follows: vpq_all:5.2423, vpq_thing:3.4507,vpq_stuff:6.8168.
